api/api.go

```diff
@@ -685,6 +685,18 @@ func (api *api) UpdateChannel(ctx context.Context, updateChannelRequest *UpdateC
 	return api.svc.GetLNClient().UpdateChannel(ctx, updateChannelRequest)
 }
 
+func (api *api) GenerateOffer(ctx context.Context, description string) (string, error) {
```

Comment:
```suggestion
func (api *api) MakeOffer(ctx context.Context, description string) (string, error) {
```

To be consistent with `MakeInvoice`